src/components/TextField/TextField.tsx

```diff
@@ -430,8 +430,10 @@ export function TextField({
     'aria-activedescendant': ariaActiveDescendant,
     'aria-autocomplete': ariaAutocomplete,
     'aria-controls': ariaControls,
-    'aria-multiline': normalizeAriaMultiline(multiline),
     'aria-expanded': ariaExpanded,
+    ...(normalizeAriaMultiline(multiline) && {
```

Comment:
Instead of doing object spread stuff would it be possible to change `normalizeAriaMultiline` so if it is undefined, it will return undefined instead of [changing to false](https://github.com/Shopify/polaris-react/blob/62c9cbb8ad45cd9e95639f862966d07222cc93c3/src/components/TextField/TextField.tsx#L530-L531)?